app/javascript/components/dashboard-form/dashboard-form.schema.js

```diff
@@ -0,0 +1,50 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+
+function createSchema(maxNameLen, maxDescLen, readOnly) {
+  const fields = [
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'name',
+      label: __('Name'),
+      maxLength: maxNameLen,
+      validate: [
+        {
+          type: validatorTypes.REQUIRED,
+          message: __('Required')
```

Comment:
No need for messages for required validator if it uses default message. Its set globally.